src/vic.h

```diff
@@ -52,7 +52,8 @@
 
 #endif
 
-#define VIC_BUFFER_SIZE 128
+#define VIC_BUFFER_IN_SIZE 64
```

Comment:
@mrshu Yes, the longest command is "set 'var name' 'var value'", that makes 30 bytes, but there may be more spaces, so I've chosen 64.